packages/voiceflow-types/src/node/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseNode, BaseText } from '@voiceflow/base-types';
+
+export type VoiceflowPrompt = BaseText.SlateTextValue | string;
+
+export interface NoMatchNode extends BaseNode.Utils.BaseNode, BaseNode.Utils.DeprecatedNodeNoMatch<VoiceflowPrompt> {
```

Comment:
this is to enable:https://github.com/voiceflow/general-runtime/blob/6cecb8fc24e7b95ad6d19e77062c32598becfa99/lib/services/runtime/handlers/noMatch.ts#L12